python/lsst/analysis/tools/actions/keyedData/stellarLocusFit.py

```diff
@@ -28,164 +28,186 @@
 import numpy as np
 import scipy.odr as scipyODR
 from lsst.pex.config import DictField
+from lsst.pipe.base import Struct
 
 from ...interfaces import KeyedData, KeyedDataAction, KeyedDataSchema, Scalar, Vector
 from ...math import sigmaMad
 
 
-def stellarLocusFit(xs, ys, paramDict):
+def stellarLocusFit(xs, ys, mags, paramDict):
```

Comment:
Is it a problem that the API is changing here?

Reply:
I'm sure it's fine.  Though if it's not used anywhere outside this Action file, maybe rename to `_stellarLocusFit`?